lara-typescript/src/interactive-api-client/README.md

```diff
@@ -0,0 +1,37 @@
+# LARA Interactive API
+
+### This documentation is meant to be used by LARA interactive developers.
+
+#### Setup and webpack configuration
+
+The API is available under `window.LARA.InteractiveAPI` object / namespace once the plugin is initialized by LARA.
+
+However, if the interactive is implemented using TypeScript, the best way to get type checking and hints in your IDE is to
+install [LARA Plugin API NPM package](https://www.npmjs.com/package/@concord-consortium/lara-interactive-api):
+
+```
+npm i --save-dev @concord-consortium/lara-interactive-api
+```
+
+Then, you need to configure [webpack externals](https://webpack.js.org/configuration/externals/), so webpack does not
+bundle the API code but looks for global `window.LARA.InteractiveAPI` object instead (and do the same for React if the
+plugin uses it).
```

Comment:
This also looks like it should be updated since we do want the API code bundled in this case.